documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md

```diff
@@ -0,0 +1,258 @@
+
+- An email campaign that is sent to the subset of the grants.gov mailing list with whom we plan to share the site.
+- A communications plan that outlines future plans for stakeholder engagement about the Simpler.Grants.gov initiative. This plan should answer the following questions:
+  - Which stakeholder groups are we planning to engage?
+  - Who is responsible for managing the relationship with each group?
+  - What is the primary channel and cadence for those communications?
+
+Both the initial email communication and the communication plan should be reviewed and approved by key stakeholders in the recurring communications meetings.
+
+### Email subscription mechanism
+
+One stretch goal for this deliverable is to adopt a service that allows site visitors to subscribe to a mailing list to receive updates about the Simpler.Grants.gov initiative. As part of this deliverable the team should answer the following questions:
+
+- Which email marketing provider will we use for email-based communications?
+- How will visitors submit their information to be added to a mailing list for Simpler.Grants.gov communications?
+
+### Definition of done
+<!-- Required -->
+
+#### Must have
+
+- [ ] Basic requirements
+  - [ ] Code is deployed to main & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] All new services have passed a security review (as needed)
+  - [ ] All new services have completed a 508 compliance review (as needed)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly (as needed)
+- [ ] Functional requirements
+  - [ ] The static site has been updated with content approved by internal stakeholders
+  - [ ] The front-end infrastructure can handle site traffic of up to 1,000 concurrent users
+  - [ ] Tickets and/or documentation have been created that describe the infrastructure changes needed to support future site traffic at the following levels:
+    - [ ] 10,000 users
+    - [ ] 100,000 users
+    - [ ] 1 million users
+  - [ ] Engineering staff are notified when the site goes down
+  - [ ] Engineering staff have been trained on a documented incident response plan that describes a set of actions to take when the site goes down
+  - [ ] A set of SLAs have been defined internally for site availability and responsiveness
+- [ ] Communications requirements
+  - [ ] An email has been sent out to the subset of public stakeholders with whom we plan to share the static site
+  - [ ] Key internal stakeholders (e.g. help desk staff, HHS leadership) have been notified when the email is sent out so that they can prepare for questions from public stakeholders
+  - [ ] A longer-term stakeholder engagement plan has been drafted that describes future phases of engagement
+  - [ ] Help Desk is notified and trained for any potential support issues that may come through
+
+#### Nice to have
```

Comment:
Same comment as above: should we add "must haves" for the list serv as well as for a survey tool for signing people up for user research?

Reply:
Sure, I added these two additional criteria in [this commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/335745fcd28577b6f0d3b40ead9d58c243fda091) and [the previous commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/bcf247f3b4986382ba7e7c64e9d411d83842b0ae)